tendrl/gluster_bridge/atoms/peer/detach.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
+
+
+class Detach(object):
+    def start(self, peer):
+        subprocess.call(['gluster', 'peer', 'detach',
```

Comment:
I think this we would take up in a separate PR and yes for sure its there back of my mind. Thanks for suggestion :)